status-im/status-desktop
# test: storenode ci

Requires:
- https://github.com/status-im/status-go/pull/3106

To test locally:
```
docker run -p 127.0.0.1:8645:8545 -p 127.0.0.1:61000:61000/tcp statusteam/nim-waku:v0.13.0 --store --rpc-address=0.0.0.0 --tcp-port=61000 --listen-address=0.0.0.0 --nat=none --nodekey=1122334455667788990011223344556677889900112233445566778899001122
```

Then start desktop like this:
```
TEST_PEER_ENR=/ip4/127.0.0.1/tcp/61000/p2p/16Uiu2HAmMGhfSTUzKbsjMWxc6T1X4wiTWSF1bEWSLjAukCm7KiHV make run
```

And create a new account.
In node_management if you execute `{"method":"wakuext_peers"}` you should see the peer `16Uiu2HAmMGhfSTUzKbsjMWxc6T1X4wiTWSF1bEWSLjAukCm7KiHV` and storenode functionality should be working.


Curiously enough, i saw the Disconnected banner, even tho there's a peer connected according to `wakuext_peers`. This makes me think that we have an error either emitting or reacting to the peer stats signal
